types/meteor/tslint.json

```diff
@@ -39,6 +39,7 @@
         "triple-equals": false,
         "typedef-whitespace": false,
         "unified-signatures": false,
-        "void-return": false
+        "void-return": false,
+        "await-promise": false
```

Comment:
> **await-promise**
> Warns for an awaited value that is not a Promise.

do I understand correctly that with this you **disable** the `await-promise` rule so disable the warning? 🤔 

if yes, could you elaborate, why? I thought it should be exactly the other way around: the rule should be enabled so that we don't miss cases of accidentally supplying non-promise return type for async operations.

if the rule is indeed disabled, then the test that you added for `dropIndexAsync` will never fail I think. or am I missing something?

Reply:
Yeah sorry, I should have left a comment on the PR.

This does disable the tslint rule, but that's because the rule is broken. There was a bug (palantir/tslint#3997), but tslint was deprecated before the bug was fixed, so as far as I can tell it will be broken forever.

Reply:
thanks for clarification!